### PR TITLE
Small documentation update

### PR DESCRIPTION
**Pull request by ehsteve**

### CODE REVIEW

File: docs/dev_guide/newcomers.rst [hunk 1]
```diff
@@ -37,8 +37,9 @@ Code
 If you would prefer to code Python instead, we have a SunPy issue list on Github `issues`_ where all the known issues with SunPy are kept.
 An issue might have a label and the `Package Novice label`_ is a good place to start.
 These are issues that have been deemed a good way to be eased into SunPy and are achievable with little understanding of the SunPy codebase.
-If you came here from the Mozilla Sprint, we have a `mozsprint label.`_
-These issues are similar in nature to the Package Novice label but with a slight difference that these issues are (we think) achievable within the timescale of a sprint, typically one day of hacking.
+We also tag issues for specific events such as the Mozilla Sprint
+(`mozsprint label.`_) or hacktoberfest. The scope of the issues should be appropriate for
```
Comment (Cadair) [inline docs/dev_guide/newcomers.rst]: @nabobalis is this actually still true?

Comment (nabobalis) [reply docs/dev_guide/newcomers.rst]: I thought we merged them into one label. I could be wrong.

File: docs/dev_guide/newcomers.rst [hunk 2]
```diff
@@ -110,6 +111,14 @@ If you see that astropy_helpers is different or missing please run ::
...
 which should fix the issue for you.
 
+Every once in a while you may have to update this submodule. This can be
+done with ::
+
+    git submodule update
+
+This will solve what git will consider a difference in the astropy_helper
+signature.
+
```
Comment (nabobalis) [inline docs/dev_guide/newcomers.rst]: This is a repeat of the block of text above. Maybe merge them into a clearer paragraph?

File: docs/dev_guide/newcomers.rst (hunk 1 above)
Comment (nabobalis) [reply docs/dev_guide/newcomers.rst]: Yeah we don't have the `mozsprint` label anymore, I think we decided to cover it under `Hacktoberfest `. I really don't several labels for the same kind of event.

**Pull request merged**